### PR TITLE
fix: Correct language for fr usage

### PR DESCRIPTION
**Pull request by Derick1530**
Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

_Change to French some of the comments inside the code snippets that were written in English_

---

## Screenshots

_[Screenshots]_
image1
image2
image3

💯

### CODE REVIEW

**Comment by juliusmarminge** [comment]
No other language have translated code snippets so it'd be weird that these would be translated?

**Comment by Derick1530** [comment]
I don't think so. I've checked other library, like create-react-app, they
do translate their code snippets!

**Comment by Yevhenii-Bakhmat** [comment]
>  I've checked other library
Julius means we didn't translate it in other translation of this docs.

**Comment by Derick1530** [comment]
I see, and I got it,  though it could've help some folks who switch to
their local language to get a better understanding.

**Comment by Yevhenii-Bakhmat** [comment]
Im personally scared of having non-english comments in my code. Tho i agree it could help. We should poll it somewhere

**Comment by joachimjusth** [comment]
good catch @Derick1530 
could you also update `www/src/pages/fr/usage/trpc.md`, please ?

@juliusmarminge we already translated in french some comments inside code snippets. (ex: `www/src/pages/fr/usage/env-variables.md` )

it's only for code snippets inside the doc, not to translate the code inside the `cli/template` folder ;)

**Review by Derick1530 — COMMENTED** [review]
@joachimjusth trpc fr code snippets just updated

File: www/src/pages/fr/usage/trpc.md [hunk 1]
```diff
@@ -277,24 +277,24 @@ const MyComponent = () => {
...
-      // Cancel outgoing fetches (so they don't overwrite our optimistic update)
+      // Annuler les récupérations sortantes (afin qu'elles n'écrasent pas notre mise à jour optimiste)
       await utils.post.list.cancel();
 
-      // Get the data from the queryCache
+      // Récupérer les données du queryCache
       const prevData = utils.post.list.getData();
 
-      // Optimistically update the data with our new post
+      // Mettez à jour les données de manière optimiste avec notre nouveau message
```
Comment (joachimjusth) [inline www/src/pages/fr/usage/trpc.md:286]: I think be can let `post`

in that case, post is a kind of thing

Comment (Derick1530) [reply www/src/pages/fr/usage/trpc.md:286]: Yeah

**Comment by Derick1530** [comment]
Requested Changes resolved @joachimjusth

**Comment by Derick1530** [comment]
> Requested Changes resolved @joachimjusth

@JohnBakhmat

**Comment by joachimjusth** [comment]
We had a discussion on Discord about this kind of modification. It'll be not possible to update all code snippet automatically (through all language) (included the comments).

@juliusmarminge I'm right?

More info: https://discord.com/channels/966627436387266600/1044891847287394325/1071518731966087260

@juliusmarminge and @c-ehrlich should we continue to translate the comments inside the code snippets?

**Comment by juliusmarminge** [comment]
> We had a discussion on Discord about this kind of modification. It'll be not possible to update all code snippet automatically (through all language) (included the comments).
> 
> 
> 
> @juliusmarminge I'm right?
> 
> 
> 
> More info: https://discord.com/channels/966627436387266600/1044891847287394325/1071518731966087260
> 
> 
> 
> @juliusmarminge and @c-ehrlich should we continue to translate the comments inside the code snippets?

I'm not in favor of this as it would make it more 'intimidating' for contributors to make any changes to other langs code blocks and I think it's better that the codeblocks are correct rather than having native comments?

**Pull request closed without merging**